locales/en/translation.json

```diff
@@ -678,9 +684,11 @@
   "ai:Worker Hosts Labels": "Worker Hosts Labels",
   "ai:Workers": "Workers",
   "ai:Workers: At least {{worker_cpu_cores}} CPU cores, {{worker_ram}} RAM, {{worker_disksize}} GB disk size for each worker ": "Workers: At least {{worker_cpu_cores}} CPU cores, {{worker_ram}} RAM, {{worker_disksize}} GB disk size for each worker ",
+  "ai:x86_64": "x86_64",
   "ai:You are approving multiple hosts. All hosts listed below will be approved to join the infrastructure environment if you continue. Make sure that you expect and recognize the hosts before approving.": "You are approving multiple hosts. All hosts listed below will be approved to join the infrastructure environment if you continue. Make sure that you expect and recognize the hosts before approving.",
   "ai:You are deleting multiple resources. All resources listed below will be deleted if you continue. This action cannot be undone.": "You are deleting multiple resources. All resources listed below will be deleted if you continue. This action cannot be undone.",
   "ai:You have opted out of formatting bootable disks on some hosts. To ensure the hosts reboot into the expected installation disk, manual user intervention may be required during OpenShift installation.": "You have opted out of formatting bootable disks on some hosts. To ensure the hosts reboot into the expected installation disk, manual user intervention may be required during OpenShift installation.",
   "ai:Your libvirt virtual machines should be configured to restart automatically after a reboot. You can check this by running:": "Your libvirt virtual machines should be configured to restart automatically after a reboot. You can check this by running:",
-  "ai:Your own NTP (Network Time Protocol) sources": "Your own NTP (Network Time Protocol) sources"
+  "ai:Your own NTP (Network Time Protocol) sources": "Your own NTP (Network Time Protocol) sources",
+  "Displaying only hosts with {{cpuArchitecture}} architecture in the table.": "Displaying only hosts with {{cpuArchitecture}} architecture in the table."
```

Comment:
missing `ai:` namespace

Reply:
CI should have catch that and fail. Can you take a look why it didnt ?

Reply:
done and fixed the validation here: https://github.com/openshift-assisted/assisted-ui-lib/pull/1747/commits/a47343440d22873f5f0a93572b849b9f7698dde7